dotnet/samples/TwentyQuestions/Config.cs

```diff
@@ -0,0 +1,23 @@
+namespace TwentyQuestions
+{
+    public class ConfigOptions
+    {
+        public string? BOT_ID { get; set; }
+        public string? BOT_PASSWORD { get; set; }
+        public OpenAIConfigOptions? OpenAI { get; set; }
+        public AzureOpenAIConfigOptions? AzureOpenAI { get; set; }
+    }
+
+    public class OpenAIConfigOptions
+    {
+        public string? ApiKey { get; set; }
+    }
+
+    public class AzureOpenAIConfigOptions
+    {
+        public string? ApiKey { get; set; }
+        public string? Endpoint { get; set; }
+        public string? ContentSafetyApiKey { get; set; }
+        public string? ContentSafetyEndpoint { get; set; }
```

Comment:
I prefer sepaparting azure content safety because it's not part of azure openai.

Reply:
Make sense. I still keep them under one `Azure` section, but use `Azure.OpenAIApiKey` and `Azure.ContentSafetyApiKey` to distinguish.